### DISCUSSION TITLE
Higher-contrast variants?

### DISCUSSION THREAD

**Discussion by escherba** [arcticicestudio/nord#232] [Help & Support]
Hi, I really like Nord colors, but wish there was a higher-contrast variant of this theme with a slightly darker background. For example, one of my favorite color schemes, [Lucius](https://github.com/jonathanfilip/lucius), provides several variants (particularly useful if one's monitor is imperfect!). Perhaps this feature already exists but I somehow missed it when reviewing code and docs -- if someone could point it to me, that would be great.

Thanks!

**Comment by frosetrain** [comment]
I too think that a higher-contrast variant or Nord would be nice. When I decrease my screen brightness at night, Nord's low contrast ratio makes it hard to see the text.

**Comment by jasonlong** [comment]
I have a fork with a slightly darker background that gives me contrast I prefer. 

https://github.com/jasonlong/nord-vim

**Comment by svengreb** [comment]
Thank you for your patience! 🙏🏼
It‘s been a while [since I had free time to focus more on Nord, and my open source projects in general](https://github.com/orgs/nordtheme/discussions/183#user-content-todays-scope), and invest time in this issue due to [work-life balance](https://github.com/orgs/nordtheme/discussions/183#work-life-balance).

I recently published the first [“Northern Post — The state and roadmap of Nord“ announcement](https://github.com/orgs/nordtheme/discussions/183) which includes all details about the plans and future of the _Nord_ project, including [the goal of catching up with the backlog](https://github.com/orgs/nordtheme/discussions/183#catching-up-on-the-backlog). This issue is part of the backlog and therefore I want to triage and process it to get one step closer to a “clean state“. Read the [announcement about reaching the “clean“ contribution triage state](https://github.com/orgs/nordtheme/discussions/202) in Nord‘s discussions for more details about the goal.

Due to the scope of this issue of not being “purely technical“, or being related to the project‘s code at all, I‘ll convert it into a discussion for [Nord‘s central community hub on GitHub](https://github.com/orgs/nordtheme/discussions). This way the repository is easier to maintain by separating conversations about the actual code while also making help requests, ideas and other contributions from users more visible and supportable by the community.
